### PR TITLE
SMW: Add CHANGELOG.md

### PR DESCRIPTION
**Pull request by PoryGone**
## What is this fixing or adding?
Adds a changelog markdown file to the SMW World.

## How was this tested?
I read it, and tossed it into a markdown renderer.

### CODE REVIEW

File: worlds/smw/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+### Features
+
+- New optional Location Checks
+	- 3-Up Moons
+	- Hidden 1-Ups
+	- Bonus Blocks
+	- Blocksanity
+		- All blocks that contain coins or items are included, with the exception of:
+		- Blocks in Top Secret Area & Front Door/Bowser Castle
+		- Blocks that are unreachable without glitches/unreasonable movement
```
Comment (nicholassaylor) [inline worlds/smw/CHANGELOG.md:15]:
```suggestion
			- Blocks in Top Secret Area & Front Door/Bowser Castle
			- Blocks that are unreachable without glitches/unreasonable movement
```
For emphasis

File: worlds/smw/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+### Quality of Life:
+
+- In-Game Indicators are now displayed on the map screen for location checks and received items
+- In-level sprites are displayed upon receiving certain items
+- The Camera Scroll unlocking is now only enabled on levels where it needs to be
+- SMW can now handle receiving more than 255 items
+- Significant World Code cleanup
+	- New Options API
+	- Removal of `orld: MultiWorld` across the world
```
Comment (nicholassaylor) [inline worlds/smw/CHANGELOG.md:42]:
```suggestion
	- Removal of `world: MultiWorld` across the world
```

**Review by nicholassaylor — CHANGES_REQUESTED** [review]
Just two small changes, one typo, one style

**Review by nicholassaylor — APPROVED** [review]
LGTM

File: worlds/smw/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
+# Super Mario World - Changelog
+
+
+## v2.0
+
+### Features
```
Comment (Exempt-Medic) [inline worlds/smw/CHANGELOG.md:6]:
```suggestion
### Features:
```

File: worlds/smw/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+- The PopTracker pack now has tabs for every level/sublevel, and can automatically swap tabs while playing if connected to the server
+
+### Bug Fixes:
+
+- Several logic tweaks/fixes
+
+
+## v1.1
+
+### Features
```
Comment (Exempt-Medic) [inline worlds/smw/CHANGELOG.md:52]:
```suggestion
### Features:
```

**Review by Exempt-Medic — CHANGES_REQUESTED** [review]
Minor suggestion. Did not do a close reading.
Sidenote: I realized that whether or not you used `:` when starting a list or an inner section is inconsistent. It's fine to leave it, I just thought I'd point it out

**Review by Exempt-Medic — APPROVED** [review]

**Review by ThePhar — APPROVED** [review]

**Pull request merged**